doc/README.md

```diff
@@ -0,0 +1,73 @@
+# SciPy Documentation
+
+## How to build the docs
+
+To build the html docs for local development, SciPy itself needs to be built so your
+environment needs to be set up for that.  For details on that, see the
+[Contributor Guide](http://scipy.github.io/devdocs/dev/contributor/contributor_toc.html#development-environment)).
+
+Also ensure to initialize and update submodules (this pulls in the SciPy Sphinx
+theme and `numpydoc`):
+```
+git submodule update --init
+```
+
+Now to build both SciPy itself and the docs, use:
+```
+python3 runtests.py --doc html
+```
+
+Alternatively, if you prefer to build SciPy and the docs separately rather
+than use `runtests.py`:
+```
+python setup.py develop  # in the root of the repo
+cd doc && make html
+```
+
+In case the SciPy version found by the above command is different from that of the
+latest commit in the repo, you will see a message like:
+```
+installed scipy 5fd20ec1aa != current repo git version '35fd20ec1a'
+```
+
+This indicates that you're likely picking up the wrong SciPy install, check
+with `python -c "import scipy; print(scipy.__file__)"`.
+
+If the build is successful, you can open it in your browser, e.g. with
+`firefox build/html/index.html`.
+
+
+## Building pdf docs or building for a release
+
+To build the pdf docs, which requires a LaTeX install and can be more fiddly
+to get to work, replace the doc build commands in the section above with:
+```
+python3 runtests.py --doc latex
+```
+or:
+```
+make latex
+make -C build/latex all-pdf
```

Comment:
Note that `--doc X` is equivalent to `make X` in the doc folder. That means you still need to build the pdf yourself when using `runtests.py`.

Reply:
ah yes, fixed. and restructured the headings a little